tests/sentry/issues/endpoints/test_organization_group_index.py

```diff
@@ -2693,6 +2699,75 @@ def test_snuba_query_title(self, mock_query):
         assert int(response.data[0]["id"]) == event1.group.id
         assert mock_query.call_count == 1
 
+    @patch.object(
+        PerformanceRenderBlockingAssetSpanGroupType,
+        "noise_config",
+        new=NoiseConfig(0, timedelta(minutes=1)),
+    )
+    @patch(
+        "sentry.issues.ingest.send_issue_occurrence_to_eventstream",
+        side_effect=send_issue_occurrence_to_eventstream,
+    )
+    @patch(
+        "sentry.search.snuba.executors.GroupAttributesPostgresSnubaQueryExecutor.query",
+        side_effect=GroupAttributesPostgresSnubaQueryExecutor.query,
+        autospec=True,
+    )
```

Comment:
I think you can use `store_search_issue` from `SearchIssueTestMixin` to avoid needing to do all of this